_returners/splunk_nebula_return.py

```diff
@@ -233,13 +234,18 @@ def __init__(self, token, http_event_server, host='', http_event_port='8088', ht
         # Defaults to SSL if flag not passed
         # Defaults to port 8088 if port not passed
 
-        if http_event_server_ssl:
-            buildURI = ['https://']
-        else:
-            buildURI = ['http://']
-        for i in [http_event_server, ':', http_event_port, '/services/collector/event']:
-            buildURI.append(i)
-        self.server_uri = ''.join(buildURI)
+        servers = http_event_server
+        if not isinstance(servers, list):
+            servers = [servers]
+        for server in servers:
+            if http_event_server_ssl:
+                buildURI = ['https://']
+            else:
+                buildURI = ['http://']
+            for i in [server, ':', http_event_port, '/services/collector/event']:
+                buildURI.append(i)
+            uri = [''.join(buildURI), '']
+            self.server_uri.append(uri)
```

Comment:
I feel like this could be simplified:

```
            if http_event_server_ssl:
                buildURI = 'https://'
            else:
                buildURI = 'http://'
            uri = [''.join([buildURI, server, ':', http_event_port, '/services/collector/event']), '']
            self.server_uri.append(uri)
```

No need to build the buildURI list with logic if the whole thing is hardcoded anyway.